sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Models/Employee.cs

```diff
@@ -0,0 +1,70 @@
+// ------------------------------------------------------------------------------
+// <auto-generated>
+//    Generated by ApacheAvroTestTool, version 1.10.0.0
+//    Changes to this file may cause incorrect behavior and will be lost if code
+//    is regenerated
+// </auto-generated>
+// ------------------------------------------------------------------------------
+namespace TestSchema
+{
+	using System;
+	using System.Collections.Generic;
+	using System.Text;
+	using Avro;
+	using Avro.Specific;
+	
+	public partial class Employee : ISpecificRecord
+	{
+		public static Schema _SCHEMA = Avro.Schema.Parse("{\"type\":\"record\",\"name\":\"Employee\",\"namespace\":\"TestSchema\",\"fields\":[{\"name\":\"Na" +
+				"me\",\"type\":\"string\"},{\"name\":\"Age\",\"type\":\"int\"}]}");
+		private string _Name;
```

Comment:
nit: use spaces

Reply:
This file is generated by Apache Avro. So, it is used for Avro serialization testing.